packages/mds-geography/types.ts

```diff
@@ -42,10 +42,15 @@ export type GeographyApiAccessTokenScopes =
 export type GeographyApiResponse<B = {}> = ApiVersionedResponse<GEOGRAPHY_API_SUPPORTED_VERSION, B> &
   ApiResponseLocalsClaims<GeographyApiAccessTokenScopes>
 
-export type GeographyApiGetGeographyResponse = GeographyApiResponse<{
+export type GeographyApiGetGeographyResponseBody = {
   data: { geographies: Geography[] | GeographySummary[] } | { geography: Geography | GeographySummary }
-}>
+}
 
-export type GeographyApiGetGeographiesResponse = GeographyApiResponse<{
+export type GeographyApiGetGeographyResponse = GeographyApiResponse<GeographyApiGetGeographyResponseBody>
```

Comment:
So the API endpoint using this can only return one Geography. Should the `geographies` field really be an option?

Reply:
I had the same question, but it's kind of unrelated to Levi's change.

Reply:
Are we going to fix this type declaration to only include a singular geography? @levi217 @janedotx @avatarneil

Reply:
@levi217 @drtyh2o I think we should, even if this PR isn't the place for it.

Reply:
@levi217 @janedotx I pushed a commit with corrected typings.